### PR TITLE
LPS-142296 Fix input behavior in toobar

### PR DESCRIPTION
**Pull request by ramonmsp**
Issue: [LPS-142296](https://issues.liferay.com/browse/LPS-142296)

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default function UpperToolbar({
 	};
 
 	const onInputBlur = () => {
-		if (selectedLanguageId) {
+		if (selectedLanguageId && definitionTitle) {
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js:52]: @ramonmsp Can we sort this statement `(definitionTitle && selectedLanguageId)`? Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by ramonmsp** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by anabuchmann** [comment]
Just started reviewing :)

**Comment by anabuchmann** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/109844

**Pull request closed without merging**